t/plugin/openid-connect6.t

```diff
@@ -0,0 +1,154 @@
+                                "ssl_verify": false,
+                                "bearer_only" : false,
+                                "timeout": 10,
+                                "introspection_endpoint_auth_method": "client_secret_post",
+                                "required_scopes": ["profile"],
+                                "introspection_endpoint_auth_method": "client_secret_post",
+                                "introspection_endpoint": "http://127.0.0.1:8080/realms/University/protocol/openid-connect/token/introspect",
+                                "set_access_token_header": true,
+                                "access_token_in_authorization_header": false,
+                                "set_id_token_header": true,
+                                "set_userinfo_header": true,
+                                "set_refresh_token_header": true,
+                                "session": {
+                                    "secret": "jwcE5v3pM9VhqLxmxFOH9uZaLo8u7KQK",
+                                    "cookie": {
+                                        "lifetime": 100
+                                    }
+                                }
+                            }
+                        },
+                        "upstream": {
+                            "nodes": {
+                                "127.0.0.1:1980": 1
+                            },
+                            "type": "roundrobin"
+                        },
+                        "uri": "/*"
+                }]]
+                )
+
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.say(body)
+        }
+    }
+--- response_body
+passed
+
+=== TEST 3: Call to route to get session
+--- config
+    location /t {
+        content_by_lua_block {
+            local http = require "resty.http"
+            local login_keycloak = require("lib.keycloak").login_keycloak
+            local concatenate_cookies = require("lib.keycloak").concatenate_cookies
+
+            local current_time = os.time()
+            local uri = "http://127.0.0.1:" .. ngx.var.server_port .. "/hello"
+
+            local res, err = login_keycloak(uri, "teacher@gmail.com", "123456")
+            if err then
+                ngx.status = 500
+                ngx.say(err)
+                return
+            end
+
+            local cookie_str = concatenate_cookies(res.headers['Set-Cookie'])
+            local parts = {}
+            for part in string.gmatch(cookie_str, "[^|]+") do
+                table.insert(parts, part)
+            end
+            local target_number = tonumber(parts[2], 10) - 100
+            -- ngx.say(target_number, current_time)
+            -- It takes time to execute the request, and there may be delays. So subtract one second.
+            if target_number == current_time or target_number -1 == current_time then
```

Comment:
How about 
```
if target_number >= current_time then
```
If the judgements are equal, this may lead to occasional errors

Reply:
@monkeyDluffy6017 Thank you for your suggestions, and these problems have been fixed.